### ISSUE TITLE
Blank state in side panel tabs

### ISSUE THREAD

**Issue by txau** [huridocs/uwazi#1179]
Some users are confused by the blank state in connections side panel when there are no connections. This could be easily solved by adding a blank state message.

We should add the blank state for others side panels as well.

**Comment by danicatalan** [comment]
We have been working on adding some blank states on the app. They still need to be implemented properly but the styling and texts are already redacted with @kjantin help.

These four blank states doesn't generate much doubt while creating them:

![search](https://user-images.githubusercontent.com/6364153/29514527-30aa09fc-8669-11e7-9cb1-24e91c12772b.png)
![toc](https://user-images.githubusercontent.com/6364153/29514526-30a7f9c8-8669-11e7-8bda-31d052ea8c21.png)
![references](https://user-images.githubusercontent.com/6364153/29514524-30a3e220-8669-11e7-81ac-d8bc7e960a38.png)
![connections](https://user-images.githubusercontent.com/6364153/29514525-30a6530c-8669-11e7-82b1-d1e37fd6ed2c.png)

On the other hand, we had the Information tab. Should we use a blank state too? I think we should in both documents and entities but I'm not convinced with the results on the sidepanel. Also the alignment with Downloads section its weird, should we align all blank states to the left?

![information](https://user-images.githubusercontent.com/6364153/29514769-00a4e690-866a-11e7-82f1-cb831623453f.png)
![entity-info](https://user-images.githubusercontent.com/6364153/29514768-00a18748-866a-11e7-891a-712aef226c5c.png)

Finally, we have the Uploads sidepanel. In my opinion this section is the most complex one to make blank states, and I'm not convinced about our current approach. We should create a separated task and think better about the user workflow to improve the experience and help them to create their first entities and upload the documents.

![uploads](https://user-images.githubusercontent.com/6364153/29514818-2ee86388-866a-11e7-8130-8e300d951eeb.png)

## TL;DR

- [ ] Add blank-state to Search tab.
- [ ] Add blank-state to TOC tab.
- [ ] Add blank-state to References tab.
- [ ] Add blank-state to Connections tab.
- [ ] Decide if we want blank-state on Information tab.
- [ ] Decide if we want blank-state on Entity viewer.
- [ ] Align blank-states to left, like Downloads section?
- [ ] Create a new task for Uploads blank states.

**Comment by danicatalan** [comment]
Add blank states on #1248.
Blank state for Uploads on #1250.
